Dockerfile.windows

```diff
@@ -1,4 +1,4 @@
-FROM alpine AS build
+FROM alpine:3.14 AS build
```

Comment:
We're using 3.12 everywhere else, is there any particular reason you went with 3.14 for Windows?